src/plugins/intel_cpu/src/emitters/plugin/x64/jit_bf16_emitters.hpp

```diff
@@ -42,7 +40,16 @@ class jit_uni_vcvtneps2bf16 : public jit_emitter {
 
         if (dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_bf16)) {
             Ymm out = Ymm(out_vec_idxs[0]);
-            h->vcvtneps2bf16(out, in);
+            Zmm bf16_max = Zmm(aux_vec_idxs[0]);
+            Zmm bf16_min = Zmm(aux_vec_idxs[1]);
+            h->uni_vmovups(bf16_max, table_val("bf16_max"));
+            h->uni_vmovups(bf16_min, table_val("bf16_min"));
+
+            Zmm truncated = Zmm(aux_vec_idxs[2]);
+            h->uni_vmaxps(truncated, in, bf16_min);
+            h->uni_vminps(truncated, truncated, bf16_max);
```

Comment:
This is not a truncation behavior, it is saturation implementation by clap with [min, max], but named truncated.

Reply:
Hi, @chenhu-wang : thanks for your comments, do you suggest use some kind of more efficient instruction set to solve this kind of problem?

Reply:
I mean "truncation" mode is to cut off lower bits directly. "saturation" mode is to clap with [min, max]. They have different result.  PR description/variable name and this implementation is not aligned. First of all, we need understand which mode we needed here and align on all platforms. By truncation, you can ref [h->uni_vandps(aux, aux, table_val("mask_truncation_word"));](https://github.com/openvinotoolkit/openvino/pull/27737/files#diff-ad949f1e4cc86bb1003070a0915e10c48bcde61d00776a223eba6fc3e58c7ed9R90-R94) in below lines.

Reply:
I got your point, based on the customer model outputs, it seems we need this kind of "saturation" mode here. 
and the new added functional tests also shows that "saturation" mode impl align with ov reference impl. Maybe I need to modify the PR description based on your suggestion.